example/34_batchnorm/batchnorm_forward_training_nhwc.cpp

```diff
@@ -572,17 +579,17 @@ int main(int argc, char* argv[])
                                                             2,
                                                             false, // don't time kernel
                                                             {128, 16, 6, 512},
-                                                            true,
-                                                            true,
+                                                            false,
+                                                            false,
                                                             averageFactor,
                                                             epsilon);
 
         pass = pass && bnorm_fwd_nhwc_test<ck::half_t, float, false>(true,
                                                                      2,
                                                                      false, // don't time kernel
                                                                      {128, 16, 3, 1024},
-                                                                     true,
-                                                                     true,
+                                                                     false,
```

Comment:
Why you change it? I thought true is better for the default parameter of Batchnorm training

Reply:
Miss-changing due to performance testing.   Fix by https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/771/commits/cb3d18a1ec532b3af878a868ad3c46b5c7287460